src/commands/tail/mod.rs

```diff
@@ -152,9 +137,25 @@ async fn enable_tailing_start_heartbeat(
     }
 }
 
+async fn print_logs(req: Request<Body>) -> Result<Response<Body>, hyper::Error> {
+    match (req.method(), req.uri().path()) {
+        (&Method::POST, "/") => {
+            let whole_body = hyper::body::to_bytes(req.into_body()).await?;
+            println!("{}", str::from_utf8(&whole_body).unwrap());
+
+            Ok(Response::new(Body::from("Success")))
```

Comment:
Is this response body used anywhere? What's the reasoning behind not using `Response::default`?

Reply:
tbh i didn't change this code from what Gabbi wrote, only re-org'd it.